scenarios/kubernetes_bazel.py

```diff
@@ -71,10 +71,15 @@ def main(args):
             res = 1
         else:
             try:
-                check(
-                    'bazel', 'run', '//:push-build', '--',
-                    '%s/%s' % (args.gcs, version)
-                    )
+                gcs_location = '%s/%s' % (args.gcs, version)
+                check('bazel', 'run', '//:push-build', '--', gcs_location)
+                artifacts = '%s/_artifacts' % os.environ['WORKSPACE']
+                try:
+                    os.mkdir(artifacts)
+                except OSError:
+                    pass
+                with open(artifacts+'/build_location.txt', 'w') as fp:
```

Comment:
Actually I think we're referring to different patterns, do you mean the string concat pattern or?
Artifacts logic is cargo culted from [`scenarios/kubernetes_heapster.py`](https://github.com/kubernetes/test-infra/blob/37b2f94bbcfd5d73dd6273a4bf0c70c8e005f2c3/scenarios/kubernetes_heapster.py#L41)  and [`scenarios/kubernetes_kubelet.py`](https://github.com/kubernetes/test-infra/blob/bcb2b32f9c6759400422283bbe60f2ac87f8152f/scenarios/kubernetes_kubelet.py#L64).